src/components/commons/AdaptiveList.tsx

```diff
@@ -0,0 +1,40 @@
+import { PropsWithChildren } from 'react'
+import classNames from 'classnames'
+
+export function AdaptiveList (props: PropsWithChildren<{ className?: string }>): JSX.Element {
+  return (
+    <div
+      data-testid='AdaptiveList'
+      className={classNames('divide-y divide-gray-200 h-full lg:border lg:rounded-lg overflow-hidden w-full border-gray-200', props.className)}
+    >
+      {props.children}
+    </div>
+  )
+}
+
+function Row (props: PropsWithChildren<{ className?: string }>): JSX.Element {
+  return (
+    <div
+      data-testid='AdaptiveList.Row'
+      className={classNames('py-3 pl-6 pr-4', props.className)}
+    >
+      {props.children}
+    </div>
+  )
+}
+
+function Cell (props: PropsWithChildren<{ name: string, className?: string }>): JSX.Element {
+  return (
+    <div className='flex items-center'>
+      <div className='w-24 lg:w-56 flex-shrink-0'>
```

Comment:
I would highly opt against defining explicit w-* or h-* in any component. It immediately makes it less responsive. A better way to design would be automatic or via min/max width.

Reply:
I have added a min-width extend to tailwind config to solve this.